core/build.gradle.kts

```diff
@@ -386,3 +411,14 @@ tasks.withType<AbstractDokkaLeafTask>().configureEach {
         }
     }
 }
+
+tasks.whenTaskAdded {
```

Comment:
Why it's required to use a callback?

Reply:
This callback allows to disable task when it added. Such tasks added lazily by kotlin.gradle plugin so I think it is safer to make it via callback.

Reply:
I believe usual task configuration would be ok.
And why do you need to disable it in the first place?

Reply:
The metadata compiler task cannot compile intermediate `js-wasm` shared sourceset (no `js-wasm stdlib`).

I tried to replace it by `getByName` but task is not exists in this script execution point so it failed.

Reply:
> The metadata compiler task cannot compile intermediate

What is the problem? Some missing dependencies?

Reply:
> I tried to replace it by getByName but task is not exists in this script execution point so it failed.

Can you get a definitive answer for that from KMP team? I was under impression that all compilation tasks are created as soon as targets and source sets are configured.

Reply:
Asking `KMP team` that does not exists anymore sounds more difficult than replace `whenTaskAdded` with `configureEach` that works as well. Is it ok for you?